### PR TITLE
test(libs-modules-element-api) add create-element-spec

### PR DESCRIPTION
**Pull request by angelwong0963**
## Current Behavior

There's no test in the libs-modules-element-api

## Expected Behavior
Make the following command to work.
`nx test modules-element-api ` 

## Related Issue(s)
None

### CODE REVIEW

File: libs/modules/element-api/src/use-cases/create-element/create-element.i.spec.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    app = await setupTestModule(app, PageModule, ElementModule)
+
+    const auth0Service = app.get(Auth0Service)
+    accessToken = await auth0Service.getAccessToken()
+
+    // Create test page
+    const createdPage = await createTestPage(app, accessToken)
+
+    // ? : please give me your advice. I want to extract non null value of page.
+    if (createdPage) {
```
Comment (webberwang) [inline libs/modules/element-api/src/use-cases/create-element/create-element.i.spec.ts:63]: if (!createdPage) {
  // throw error
}

I would have done that

Comment (angelwong0963) [reply libs/modules/element-api/src/use-cases/create-element/create-element.i.spec.ts:63]: Ok

File: libs/modules/element-api/src/use-cases/create-element/create-element.i.spec.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import { createTestElement } from '../../utils/testHelpers'
+
+describe('CreateElement', () => {
+  let app: INestApplication
+  let accessToken = ''
+
+  beforeAll(async () => {
+    app = await setupTestModule(app, ElementModule)
+
+    const auth0Service = app.get(Auth0Service)
```
[the remark this answers is not in the document]
Comment (angelwong0963) [reply libs/modules/element-api/src/use-cases/create-element/create-element.i.spec.ts:25]: Sure, I rebased this branch on top of the latest mater locally. I am resolving the conflicts. I will use mocking for all the specs from now on.

**Comment by webberwang** [comment]
@angelwong0963 followed the project coding style very nicely!

**Pull request merged**